tasks/xgettext.js

```diff
@@ -27,13 +27,15 @@ module.exports = function(grunt) {
 
   /**
    * Get all messages of a content
-   * @param  {String} content     content on which extract gettext calls
-   * @param  {Regex} regex        first level regex
-   * @param  {Regex} subRE        second level regex
-   * @param  {Regex} quoteRegex   regex for quotes
-   * @param  {String} quote       quote: " or '
-   * @param  {Object} options     task options
-   * @return {Object}             messages in a JS pot alike
+   * @param  {String} content           content on which extract gettext calls
```

Comment:
Either indent all of these the same distance or (preferably) use a single space between argument and description.